src/components/learner-credit-management/RequestRecentActionTableCell.jsx

```diff
@@ -3,13 +3,26 @@ import PropTypes from 'prop-types';
 
 const RequestRecentActionTableCell = ({ row }) => {
   const { original } = row;
-  const { requestDate, requestStatus } = original;
+  const {
+    requestDate,
+    requestStatus,
+    latestAction,
+    lastActionDate,
+  } = original;
 
-  // For approved requests, the recent action is typically the approval
-  const formattedActionType = `${requestStatus.charAt(0).toUpperCase()}${requestStatus.slice(1)}`;
+  const formatRequest = () => {
+    const hasRemindedAction = latestAction && latestAction.recentAction === 'Reminded';
```

Comment:
I would suggest you use some constants for comparison instead of a hardcoded value